examples/experimental/i3s-17-and-debug/components/debug-panel.js

```diff
@@ -0,0 +1,154 @@
+import styled from 'styled-components';
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
```

Comment:
Fine if you feel it is worth it, but generally try to avoid bringing in too many dependencies.